examples/quick-start/bentoml-quick-start-guide.ipynb

```diff
@@ -15,11 +15,62 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {
     "scrolled": true
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
```

Comment:
let's clear the output cells of this notebook - we plan to just keep output cells for the notebooks in gallery repo

Reply:
Yes sure